mne/viz/_figure.py

```diff
@@ -1854,6 +1863,22 @@ def _update_yaxis_labels(self):
                    else 'normal')
             text.set_style(sty)
 
+    def _xtick_timestamp_formatter(self, _, xval):
```

Comment:
What does the `_` parameter do?

Reply:
One of the two is the position on the axis while the other one is the displayed value. Thanks, I just realised that I mixed those two and time was not showing correctly.

Reply:
The first should be the value and the second the position